src/run/run_test.go

```diff
@@ -26,10 +26,19 @@ func TestRun_Exec_UnknownExecutable(t *testing.T) {
 }
 
 func TestRun_Exec_ExitCode(t *testing.T) {
-	_, err := run.Exec("bash", "-c", "exit 2")
-	var execError *exec.ExitError
-	assert.True(t, errors.As(err, &execError))
-	assert.Equal(t, 2, execError.ExitCode())
+	result, err := run.Exec("bash", "-c", "echo 'hi' && exit 2")
+	assert.Equal(t, 2, result.ExitCode())
+	expectedError := `
+----------------------------------------
+Diagnostic information of failed command
+
+Command: bash -c echo 'hi' && exit 2
```

Comment:
This should be okay without the single quotes since `hi` contains no quotable characters.

Reply:
Removed quotes